internal/core/config.go

```diff
@@ -104,6 +111,14 @@ type ServiceConfiguration struct {
 	CommitmentBehaviorPerResource regexpext.ConfigSet[liquid.ResourceName, CommitmentBehavior] `yaml:"commitment_behavior_per_resource"`
 }
 
+type PrometheusCapacityConfiguration struct {
```

Comment:
I generally like to have a docstring on all public fields, to make documentation pages like https://pkg.go.dev/github.com/sapcc/limes/internal/core useful. For types like this, I usually go with just the minimum amount of breadcrumbs:

```suggestion
// PrometheusCapacityConfiguration appears in type LiquidConfiguration.
type PrometheusCapacityConfiguration struct {
```